packages/react-devtools-shared/src/devtools/views/Profiler/CommitFlamegraph.js

```diff
@@ -123,10 +131,22 @@ function CommitFlamegraph({chartData, commitTree, height, width}: Props) {
     return null;
   }, [chartData, selectedFiberID, selectedChartNodeIndex]);
 
+  const handleElementMouseEnter = ({id, name}) => {
+    highlightNativeElement(id); // Highlight last hovered element.
+    setHoveredFiberData({id, name}); // Set hovered fiber data for tooltip
+  };
+
+  const handleElementMouseLeave = () => {
+    clearHighlightNativeElement(); // clear highlighting of element on mouse leave
+    setHoveredFiberData(null); // clear hovered fiber data for tooltip
+  };
+
   const itemData = useMemo<ItemData>(
     () => ({
       chartData,
-      hoverFiber,
+      isHovered: hoveredFiberData !== null,
```

Comment:
This isn't that useful.

We could fix it by passing in the hovered fiber ID, but that would require us to invalidate our memoization every time the mouse moved over something new which...seems like it's not worth the added benefit. I'm going to just suggest removing this.